### ISSUE TITLE
Why is the training performance of my VPT so poor?

### ISSUE THREAD

**Issue by Jolaris** [KMnP/vpt#62]
I trained a model for skin disease image binary classification using a pre-trained ViT model with VPT. However, after training, my accuracy only reached 60%, and the loss remained at 0.6, unable to further increase accuracy. Why is this happening? I am a beginner in deep learning. Please give me some advice. Thank you!

**Comment by GrantorShadow** [comment]
could you give more details? maybe number of tokens used? shallow or deep prompts? any more info?

**Comment by LaughAwayZHY** [comment]
> could you give more details? maybe number of tokens used? shallow or deep prompts? any more info?

Hello, have you found the corresponding hyperparameter link?
